### PR TITLE
Fixed some broken links

### PR DESCRIPTION
**Pull request by mushlih-almubarak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,8 +116,8 @@ To use the GitHub Action with Codacy integration:
...
-    -   **If you're setting up multiple repositories**, [obtain an account API token](../codacy-api/api-tokens/#account-api-tokens) and store it as an [encrypted secret for your **organization**](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization) with the name `CODACY_API_TOKEN`.
+    -   **If you're setting up one repository**, [obtain a project API token](https://github.com/codacy/docs/blob/master/docs/codacy-api/api-tokens.md#generating-and-revoking-project-api-tokens--idproject-api-tokens) and store it as an [encrypted secret for your **repository**](https://docs.github.com/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `CODACY_PROJECT_TOKEN`.
+    -   **If you're setting up multiple repositories**, [obtain an account API token](https://github.com/codacy/docs/blob/master/docs/codacy-api/api-tokens.md#generating-and-revoking-account-api-tokens--idaccount-api-tokens) and store it as an [encrypted secret for your **organization**](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization) with the name `CODACY_API_TOKEN`.
```
Comment (prcr) [inline README.md]: The links should point to the rendered pages on the Codacy documentation site instead:

```suggestion
    -   **If you're setting up one repository**, [obtain a project API token](https://docs.codacy.com/codacy-api/api-tokens/#project-api-tokens) and store it as an [encrypted secret for your **repository**](https://docs.github.com/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) with the name `CODACY_PROJECT_TOKEN`.
    -   **If you're setting up multiple repositories**, [obtain an account API token](https://docs.codacy.com/codacy-api/api-tokens/#account-api-tokens) and store it as an [encrypted secret for your **organization**](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-an-organization) with the name `CODACY_API_TOKEN`.
```

**Review by prcr — CHANGES_REQUESTED** [review]
@mushlih-almubarak thanks for spotting the broken links and proposing a fix!

Your pull request includes changes to some images as well. Are these changes intended? :thinking: Could you ensure that your pull request only includes the fixed links? Thanks!

**Comment by mushlih-almubarak** [comment]
Ah sorry, @prcr, that's a commit automatically by [this app](https://github.com/marketplace/imgbot) to compress the images in the repository, and it looks like I messed up the commit.

Because of that, I deleted your forked repository in my account and then forked it again, and now I've made changes to fix the broken link and opened this pr #77.

**Comment by prcr** [comment]
No problem, @mushlih-almubarak - we also use a similar [image compression app](https://github.com/codacy/docs/blob/master/.github/workflows/calibre.yml) on the main docs repository, but running once per week. :+1:

I'm going to review the new pull request now.

**Pull request closed without merging**